quaterion/loss/multi_objective_loss.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
```

Comment:
This is a good start, but it has some issues:

- Not well documented with docstrings.
- Argument `groups` is missing in the `forward()` method.
- Tests are missing.

After fixing these issue it might be a good start, but I'm thinking of something slightly different that will make `GroupLoss` and `PairLoss` compatible with one another through some utility functions. For example, I got slightly better results when I combined Triplet Loss and Multiple Negatives Ranking Loss, but it will require more experiments.

tl; dr: Let's remove `MultiObjectiveLoss` from this PR for now --it will be something more ambitious and definitely is worth its own PR 😁

Reply:
Sure! I will update the PR with the CosFace Loss only. 

As for the issues with the Multi Objective Loss, I will surely solve them and put a new PR, and we can experiment with it there.

Thankyou!

Reply:
I have done the required changes: 
- Removed the Multi Objecive Loss 
- Resolved the conflicts